instance/templates/instance/ansible/vars.yml

```diff
@@ -83,5 +83,5 @@ EDXAPP_WORKER_DEFAULT_STOPWAITSECS: 1200
 COMMON_ENABLE_NEWRELIC: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENABLE_NEWRELIC_APP: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENVIRONMENT: opencraft
-COMMON_DEPLOYMENT: '{{ instance.sub_domain }}'
+COMMON_DEPLOYMENT: '{{ instance.internal_lms_domain }}'
```

Comment:
@mtyaka I see - using the full domain is even better (more informative) then. Great :)